src/elements/content-sidebar/ActivitySidebar.js

```diff
@@ -447,7 +449,9 @@ class ActivitySidebar extends React.PureComponent<Props, State> {
                     isDisabled={isDisabled}
                     onCommentCreate={this.createComment}
                     onCommentDelete={this.deleteComment}
-                    onTaskCreate={this.createTask}
+                    onTaskCreate={
+                        getProp(features, 'activityFeed.tasks.createButton', false) ? this.createTask : undefined
```

Comment:
I think we should use the `isFeatureEnabled` util here, but more importantly, read the feature at the point of need to disable the checkbox rather than nulling out the callback and adding conditions for it.